tests/VarDumperTest.php

```diff
@@ -420,6 +488,7 @@ public function testExportWithClosureArray(): void
 
     public function testExportClosureWithAnImmutableInstanceOfClosureExporter(): void
     {
+        self::markTestSkipped('Must be reviewed');
```

Comment:
???

Reply:
> ???

It produces an error. I don't now for what purpose we need a static private cache of CosureExporter at all.

Reply:
It does not produce any error for me.

Reply:
> It does not produce any error for me.

Try older PHP 7.4 version
https://3v4l.org/ScB6P

Reply:
That was a bug: https://bugs.php.net/bug.php?id=69804 Not sure we need to care about it.